extensions/common/auth/auth-delegated/src/test/java/org/eclipse/edc/api/auth/delegated/DelegatedAuthenticationExtensionTest.java

```diff
@@ -57,4 +73,34 @@ public void delegatedProvider(DelegatedAuthenticationExtension extension) {
         verify(config).getLong(AUTH_CACHE_VALIDITY_MS, DEFAULT_CACHE_TIME_TO_LIVE);
 
     }
+
+    @Test
+    void initializeWithAudience(ServiceExtensionContext context, ObjectFactory factory) {
```

Comment:
instead of using the objectfactory, you can directly inject the `DelegatedAuthExtension` into the test method, c.f. the previous test method

Reply:
This is what I have done at first but if I do that the audience value is already injected with a null value before I can set it in the test.

Reply:
ah, that's right. sorry, overlooked that.